data/korath/korath outfits.txt

```diff
@@ -652,7 +652,141 @@ outfit "Steering (Stellar Class)"
 		"frame rate" 6
 	"steering flare sound" "plasma huge"
 	description "After millennia of war, the Korath have developed some truly enormous warships. These steering systems give even their largest warships a reasonable amount of agility in combat."
-	
+
+outfit "Arkrof GP Hybrid Thruster"
+	plural "Arkrof GP Hybrid Thrusters"
+	category "Engines"
+	"cost" 165000
+	thumbnail "outfit/tiny efreti thruster"
+	"mass" 17
+	"outfit space" -17
+	"engine capacity" -17
+	"thrust" 13
+	"thrusting energy" 2.1
+	"thrusting heat" 2.4
+	"flare sprite" "effect/efreti flare/tiny"
+		"frame rate" 8
+	"flare sound" "ion tiny"
+	description "Forced to abandon their beloved jump drive technology, Efreti engineers have begun to grudgingly take inspiration from their not-entirely-welcome Quarg guardians. The incorporation of gravitational propulsion increases the thrust output for engines in this line, at the cost of even higher energy use. These are the smallest such gravitoplasma-based thrusters."
+
+outfit "Farves GP Hybrid Thruster"
+	plural "Farves GP Hybrid Thrusters"
```

Comment:
Do we really need this? iirc the game already add `s` at the end of the name for plural. It's really used only in special cases like `es` or something else.

Reply:
```suggestion
```